src/hooks/useBulkPermissions.js

```diff
@@ -21,6 +21,11 @@ export const useBulkPermissions = () => {
   const hasInstancePerms = stripes.hasPerm('ui-inventory.instance.edit');
   const hasInventoryInstanceViewPerms = stripes.hasPerm('ui-inventory.instance.view');
 
+  // Quick marc
+
+  const hasQuickMarcEditPerms = stripes.hasPerm('ui-quick-marc.quick-marc-editor.all');
```

Comment:
Please note that this permission is for editing MARC Bibliographic records only. If you need to check for all types (MARC Holdings and MARC Authorities) then you should also include `ui-quick-marc.quick-marc-holdings-editor.all` and `ui-quick-marc.quick-marc-authorities-editor.all`
Same applies to viewing permissions - we have separate permissions for separate record types

Reply:
We need only  `Bibliographic records` perms regarding our description story.